src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -81,22 +86,174 @@ def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai):
         ),
     )
 
-    # Always Validation
-    always_validation_sequence_set = [[]]
+    # Placement Always Validation
+    placement_always_validation_sequence_set = [[]]
 
-    # Eventually Validation
-    eventually_validation_sequence_set = [
+    # Placement Eventually Validation
+    placement_eventually_validation_sequence_set = [
         [
             # Ball should arrive within 5cm of placement point
-            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.05)]),
+            BallEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
+            RobotEventuallyEntersRegion(regions=[tbots.Circle(ball_final_pos, 0.1)]),
         ]
     ]
 
     simulated_test_runner.run_test(
-        eventually_validation_sequence_set=eventually_validation_sequence_set,
-        always_validation_sequence_set=always_validation_sequence_set,
-        test_timeout_s=15,
+        eventually_validation_sequence_set=placement_eventually_validation_sequence_set,
+        always_validation_sequence_set=placement_always_validation_sequence_set,
+        test_timeout_s=TEST_DURATION,
+    )
+
+    # TODO (#2797): uncomment tests below to verify robots actually drop ball and exit region
+    # send a free kick command
+    # simulated_test_runner.gamecontroller.send_ci_input(
+    #     gc_command=Command.Type.DIRECT_FREE_BLUE, team=Team.BLUE
+    # )
```

Comment:
Im a bit confused by this commented out code. Would this be part of the above test? If not, then it doesn't seem like this is a complete test implementation (doesn't position the robots or override the play that's suppose to run...)

Reply:
it is a continuation of the previous test. If the previous passed and validation finished, we move to the second stage with the same world positions

Reply:
Interesting, I didn't realize we can call `run_test` multiple times in the same test